pkg/autoscaler/autoscaler/util.go

```diff
@@ -91,11 +93,54 @@ func limitTargetReplicas(targetReplicas int32, tac *v1alpha1.TidbClusterAutoScal
 // If the minReplicas not set, the default value would be 1
 // If the Metrics not set, the default metric will be set to 80% average CPU utilization.
 // defaultTAC would default the omitted value
-func defaultTAC(tac *v1alpha1.TidbClusterAutoScaler) {
+func defaultTAC(tac *v1alpha1.TidbClusterAutoScaler, tc *v1alpha1.TidbCluster) {
```

Comment:
Suggest adding some basic validation, e.g. min<max<1

Reply:
what should we do if the validating failed, e.g. min > max

Reply:
We can move the defaulting logic from https://github.com/pingcap/tidb-operator/blob/master/pkg/autoscaler/autoscaler/autoscaler_manager.go#L110 to the function https://github.com/pingcap/tidb-operator/blob/master/pkg/autoscaler/autoscaler/autoscaler_manager.go#L70, after the defaulting logic, call the validation function, if the validation fails, just print the detailed error and return nil, no need to retry the sync for this autoscaler CR.

Reply:
BTW, could you please also help remove the logic https://github.com/pingcap/tidb-operator/blob/master/pkg/autoscaler/autoscaler/autoscaler_manager.go#L89-L97, it's not necessary to update and check the TidbCluster status here.